Sources/SwiftGRPCNIO/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -67,6 +76,16 @@ extension HTTP1ToRawGRPCServerCodec: ChannelInboundHandler {
       assert(state.expectingBody, "received body while in state \(state)")
       buffer.write(buffer: &body)
 
+      // If the contentType is text, then decode the incoming data as Base64, and continue
+      // processing normally after decoding.
+      if contentType == .text {
```

Comment:
I think this will break down if the request is chunked (and I think all HTTP clients will start chunking for sufficiently large HTTP requests; you might want to add a test for that). Example:

1. First chunk is 400 bytes Base64, gets decoded to 300 bytes binary.
2. Second chunk (also Base64) arrives, is appended to the 300 binary bytes. The result is half-binary (first chunk), half-base64 (second chunk).

Let me know if I am missing something.

Reply:
I'll have to test more, haven't thought of a chunked mechanism, will keep open for now

Reply:
Might be necessary to have two buffers; one for Base64 input and another one for the already-decoded (binary) parts.